frontend/.eslintrc.json

```diff
@@ -1,40 +0,0 @@
-{
-  "env": {
-    "browser": true,
-    "es2021": true
-  },
-  "ignorePatterns": [
-    "*.js",
-    "*.config.js",
-    "dist/**/*",
-    "**/*.css",
-    "**/*.scss",
-    "**/*.md"
-  ],
```

Comment:
many of these didn't seem to do anything - I'm only excluding the static builds in `build/` and `storybook-static/` and nothing appears to trip otherwise :shrug: